.travis.yml

```diff
@@ -3,6 +3,7 @@ dist: bionic
 env:
   global:
     - VAGRANT_VERSION="2.2.7"
+    - BASE_BOX_VAGRANT_VM_ID="base-box-builder.k8s-play.local"
```

Comment:
testbed name is configurable in defaults.yaml
`playground_name: k8s-play`

Reply:
fixed. Set a value in the env.yaml that we generate for travis